<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>查询</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="button_input_com">
                <property name="text">
                 <string>导入查询公司(.txt)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" rowspan="2">
               <widget class="QTextBrowser" name="browser_com">
                <property name="tabStopWidth">
                 <number>60</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="button_more_search">
                <property name="text">
                 <string>批量查询</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_one_com">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_com">
              <property name="placeholderText">
               <string>请输入公司名称...（至少4个字符）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_one_search">
              <property name="text">
               <string>单个查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="Line" name="line_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>控制台：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>数据</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableWidget" name="table">
          <column>
           <property name="text">
            <string>序号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>企业名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>社会统一信用代码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>法定代表人</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>登记状态</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>成立日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>注册资本</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>实缴资本</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>核准日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>组织机构代码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>工商注册号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>纳税人识别号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>企业类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>营业期限</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>纳税人资质</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>所属行业</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>所属地区</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>登记机关</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>人员规模</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>参保人数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>曾用名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>英文名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>进出口企业代码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>注册地址</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>经营范围</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="button_output_csv">
            <property name="text">
             <string>导出数据为csv</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_output_excel">
            <property name="text">
             <string>导出数据为excel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>配置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>企查查账号密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>账号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_account"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>密码：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_pw"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>74</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_renew_user">
            <property name="text">
             <string>更新账号密码 -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>74</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>当前使用</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_account">
              <property name="text">
               <string>账号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_pw">
              <property name="text">
               <string>密码：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>代理：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_agent">
              <property name="placeholderText">
               <string>请在此处输入代理...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>cookie：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_cookie">
              <property name="placeholderText">
               <string>请在此处输入cookie...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_renew_setting">
            <property name="text">
             <string>更新代理与cookie -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>当前使用</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="browser"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
